recipes-kernel/linux/linux-fslc-imx_5.15.bb

```diff
@@ -70,18 +70,18 @@ include linux-fslc.inc
 LICENSE = "GPL-2.0-only"
 LIC_FILES_CHKSUM = "file://COPYING;md5=6bc538ed5bd9a7fc9398086aedcd7e46"
 
-KBRANCH = "5.15-1.0.x-imx"
-SRCREV = "b6597629c15cf4fc99dbcdaa079cb1b4c3e92f2e"
+KBRANCH = "5.15-2.0.x-imx"
+SRCREV = "8fe4fe4ac6ffc1510f98e3776895bbc1c07329a8"
 
 # PV is defined in the base in linux-imx.inc file and uses the LINUX_VERSION definition
 # required by kernel-yocto.bbclass.
 #
 # LINUX_VERSION define should match to the kernel version referenced by SRC_URI and
 # should be updated once patchlevel is merged.
-LINUX_VERSION = "5.15.48"
+LINUX_VERSION = "5.15.40"
```

Comment:
48 -> 40?

Reply:
the branch for 2.0.0 right now is at 40. I'm working to get it on later versions, but wanted to review the PR to test the kernel so far, @thochstein